Makefile

```diff
@@ -1,7 +1,16 @@
-.PHONY: deps test
+.PHONY: deps docker prepare rel test
 
 deps:
 	mix deps.get
 
+docker:
+	docker build . -t breaking-pp
+
+prepare:
+	mix local.hex --force
```

Comment:
You also need `make local.rebar --force` here, otherwise `make rel` fails in when building Docker image.

Reply:
Hmm, does it crash for you? I believe we don't use any rebar dependency for now (cowboy uses erlang.mk)

Reply:
Yup, I have an error when ranch is being compiled.

Reply:
Which is strange, because it uses erlang.mk as well.

Reply:
Got it, it copied compiled beams from my local directory, so I also added a cleanup step, to make sure we compile everything from scratch in Docker.

Can you try now?

Reply:
Works well now 👍

Reply:
Thanks! Good catch!